import "mapreduce/yt/interface/protos/extension.proto";

message TL7AccessLogRow {
    optional string IpPort = 1 [(NYT.column_name) = "ip_port"];
    optional string Workflow = 2 [(NYT.column_name) = "workflow"];
}
